### PR TITLE
[Application] Fix widget instance id bug

### PR DESCRIPTION
**Pull request by myroot**
### Description of Change ###

 Fix https://github.com/Samsung/TizenFX/issues/1150

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.Applications.WidgetApplication/Tizen.Applications/WidgetType.cs [hunk 1]
```diff
@@ -52,7 +52,7 @@ private int OnCreate(IntPtr context, IntPtr content, int w, int h, IntPtr userDa
             if (b == null)
                 return 0;
 
-            b.Bind(context, Id);
+            b.Bind(context, Interop.Widget.GetAppId(context));
```
Comment (hjhun) [inline src/Tizen.Applications.WidgetApplication/Tizen.Applications/WidgetType.cs]: GetAppId() => GetInstanceID()

Comment (myroot) [reply src/Tizen.Applications.WidgetApplication/Tizen.Applications/WidgetType.cs]: First, interop method naming is following native API naming and no api `get_instance_id` on appcore-widget
Second `GetAppId` is `widget_app_get_id` native API

``` c
367EXPORT_API const char *widget_app_get_id(widget_context_h context)
368{
369	int ret;
370	char *id;
371
372	if (!context) {
373		set_last_result(WIDGET_ERROR_INVALID_PARAMETER);
374		return NULL;
375	}
376
377	ret = widget_base_context_get_id((widget_base_instance_h)context, &id);
378	if (ret != WIDGET_BASE_ERROR_NONE) {
379		_E("failed to get context id"); /* LCOV_EXCL_LINE */
380		set_last_result(ret); /* LCOV_EXCL_LINE */
381		return NULL; /* LCOV_EXCL_LINE */
382	}
383
384	set_last_result(WIDGET_ERROR_NONE);
385	return id;
386}
```
It call `widget_base_context_get_id` native API

``` c
1093EXPORT_API int widget_base_context_get_id(widget_base_instance_h context, char **id)
1094{
1095	appcore_multiwindow_base_instance_h instance_h;
1096
1097	if (!__is_widget_feature_enabled()) {
1098		LOGE("not supported"); /* LCOV_EXCL_LINE */
1099		return WIDGET_BASE_ERROR_NOT_SUPPORTED; /* LCOV_EXCL_LINE */
1100	}
1101
1102	instance_h = (appcore_multiwindow_base_instance_h)context;
1103	*id = (char *)appcore_multiwindow_base_instance_get_id(instance_h);
1104
1105	return WIDGET_BASE_ERROR_NONE;
1106}
1107
```
and it return a result value of `appcore_multiwindow_base_instance_get_id`. 
it seems to return instance id

Comment (myroot) [reply src/Tizen.Applications.WidgetApplication/Tizen.Applications/WidgetType.cs]: I update, Interop method namaing to same with native api

Comment (hjhun) [reply src/Tizen.Applications.WidgetApplication/Tizen.Applications/WidgetType.cs]: Okay. Thanks.

**Pull request merged**